tests/fread/test-fread-large.py

```diff
@@ -106,6 +106,25 @@ def f(request):
 # Run the tests
 #-------------------------------------------------------------------------------
 
+
+# Make sure we can read/write csv and jay files larger than 4Gb on Windows
+# @pytest.mark.usefixtures("winonly", "is_release")
+@pytest.mark.usefixtures("winonly")
+def test_fread_4gb_plus(tempfile):
+    size = 5 * 10**6
+    DT0 = dt.Frame([True] * size)
+    DT0.to_jay(tempfile)
+    assert(os.path.getsize(tempfile) > size)
+
+    DT1 = dt.Frame(tempfile)
+    assert_equals(DT0, DT1)
+    DT1.to_csv(tempfile)
+    assert(os.path.getsize(tempfile) > size)
```

Comment:
You're asserting only that file size is greater than 5Mb.

Reply:
This is WIP, so I want to test 5MB first then see what happens at larger sizes...